### PR TITLE
Bug 1210304 - Renamed pinApp to pinnedApp

### PR DESCRIPTION
**Pull request by MariaKolesina**

### CODE REVIEW

**Comment by MariaKolesina** [comment]
@harman-red-square/harman-reviewers

**Comment by ruvsmirnov** [comment]
It looks good for me

File: apps/verticalhome/js/app.js [hunk 1]
```diff
@@ -165,16 +166,16 @@
...
 
-    getPinAppList: function() {
-      return this.itemStore.getPinAppList();
+    getPinnedAppsList: function() {
+      return this.itemStore.getPinnedAppsList();
     },
 
-    savePinAppItem: function(obj) {
-      this.itemStore.savePinAppItem(obj);
+    savePinnedAppItem: function(obj) {
```
Comment (ruvsmirnov) [inline apps/verticalhome/js/app.js:173]: May be it would be better to name savePinnedItem or savePinnedApp

**Comment by ruvsmirnov** [comment]
Does it work good after full rebuild?

**Comment by MariaKolesina** [comment]
Yes, it works good, but first time it's required to remove folders 'out' and 'objdir-gecko'

**Comment by ruvsmirnov** [comment]
It would be better to land it first.

**Comment by egorlevichev** [comment]
:shipit:

**Pull request merged**